oe_starter_content.info.yml

```diff
@@ -9,8 +9,3 @@ dependencies:
   - drupal:node
   - drupal:datetime_range
   - drupal:address
```

Comment:
We actually don't have any fields in here anymore, so there are no dependencies to add.
They need to be moved to the related submodules now.